### DISCUSSION TITLE
Notifications on Livewire Components

### DISCUSSION THREAD

**Discussion by tomsgad** [filamentphp/filament#3119] [Help]
Hi guys,

How do I use notifications on the livewire component, I have tried using facade but with no success.

```php
public function add()
    {
        //

        $this->notify('success', 'Saved');
    }
```

No notification appears

**Comment by NathanaelGT** [comment]
`$this->notify(...)` is not a facade

try this:
```php
use Filament\Facades\Filament;

public function add()
{
    //

    Filament::notify('success', 'Saved');
}
```

**Reply by tomsgad** [reply]
Yes, i have used that facade too, its not working
